### PR TITLE
object neuron

### PR DESCRIPTION
**Pull request by LEEDASILVA**

### CODE REVIEW

File: exercise/shallow.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+## shallow
+
+### Instructions
+
+Create a function called `shallow` that allows you to copy the values of all properties from one or more
+objects to a target object.
+
+### Notions
+
+- https://devdocs.io/javascript/global_objects/object/assign
```
Comment (kigiri) [inline exercise/shallow.js]: I think this exercise is too easy.

**Review by kigiri — CHANGES_REQUESTED** [review]
Either we find a way to make it more interesting or remove it.

We could try to work with the fact that object.assign change the first given object.

You can use that to do fun stuff, maybe we could work on that.

Or we could make it so you just have to clone the given object, this way you have at least 2 simple solutions:
```js
const clone = src => Object.assign({}, src)
const clone = src => ({ ...src })
```
but it still seem easy, maybe merge it with the deep one, this could be like the 1st function of the deep one

File: exercise/shallow.js (hunk 1 above)
Comment (LEEDASILVA) [reply exercise/shallow.js]: maybe i could merge this exercise with replicate and create a new exercise.

maybe soup letter:
```js
    const grid = [
      'jefblpepre',
      'camdcimgtc',
      'oivokprjsm',
      'phelloroua',
      'rixieelhrs',
      'wolcqereht',
      'screeaumgr',
      'alxhpburyi',
      'jalaycalmp',
      'cloperermt',
    ]
// result
{
  hello: {
    start: [3, 1],
    end: [3, 6],
  },
  there: {
    start: [5, 10],
    end: [5, 6],
  },
}
```
let me now what you think

Comment (kigiri) [reply exercise/shallow.js]: yeah ok, it's more an algorithmic exercise than a simple data shape thing, but it has some object output, could be interesting.
I just feel like it's fairly hard to do it with the `map`, `filter`, i'll try.

**Review by kigiri — APPROVED** [review]

**Pull request merged**